packages/pds/src/db/migrations/20230210T210132396Z-post-hierarchy.ts

```diff
@@ -41,11 +41,15 @@ export async function up(
     })
     .selectFrom('hierarchy')
 
-  await db
-    .insertInto('post_hierarchy')
-    .columns(['uri', 'ancestorUri', 'depth'])
-    .expression(postHierarchyQb.select(['uri', 'ancestorUri', 'depth']))
-    .execute()
+  const res = await postHierarchyQb.selectAll().limit(1).execute()
+
+  if (res.length > 0) {
```

Comment:
Did some kind of error pop out of this?

Reply:
oh yup. but i think it was a previous boneheaded mistake that i think was causing it. think i can remove it now